Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/SceneUnderstandingObserverProfile.cs

```diff
@@ -19,22 +19,22 @@ public class SceneUnderstandingObserverProfile : BaseSpatialAwarenessObserverPro
         #region IMixedRealityOnDemandObserver settings
 
         [SerializeField]
-        [Tooltip("Observer will update once after initialization then require manual update thereafter.")]
-        private bool updateOnceOnLoad = false;
+        [Tooltip("Whether the observer updates once after initialization (regardless whether AutoUpdate is true).")]
+        private bool updateOnceInitialized = false;
```

Comment:
We should add `FormerlySerializedAs` tags to all the serialized fields, in case anybody is already using this